x/logic/predicate/bank_test.go

```diff
@@ -36,6 +36,7 @@ import (
 )
 
 func TestBank(t *testing.T) {
+	bench32DecodingFail := strings.Join(strings.Split("decoding bech32 failed: invalid bech32 string length 3", ""), ",")
```

Comment:
You should simplify this. The idea proposed in the GitHub issue #771, as flagged by Dependabot, was to remove string processing and directly insert the final string result. While this may reduce readability, I tend to agree with the approach because it directly represents the content we get, simplifying maintenance.

For instance:

```suggestion
	bench32DecodingFail := "d,e,c,o,d,i,n,g, ,b,e,c,h,3,2, ,f,a,i,l,e,d,:, ,i,n,v,a,l,i,d, ,b,e,c,h,3,2, ,s,t,r,i,n,g, ,l,e,n,g,t,h, ,3"
```

So basically, wherever you see `strings.Join` + `strings.Split` on a constant string in the tests, just replace it with the final result.

Reply:
I apologize for the misunderstanding. I've replaced it with a constant string literal without using any string methods. Thank you for your review.